### ISSUE TITLE
Delete redundant stuff from the repo

### ISSUE THREAD

**Issue by Hume2** [SuperTux/addons#14]
Since the addons are in a Git repo, we don't have to store bazilion out-of-date versions in .zip. If any of those were needed, it's possible to create them from the source history. Therefore we can delete the unused versions from "repository/". The addons in the folders "Worldmaps*/" are dupe and can be deleted.

We can delete the folder "Levels/", since it contains only two out-of-date addons, those are already stored somewhere else.

**Comment by Grumbel** [comment]
Careful with that, some of those are still referenced in the .nfo files and need to stay (e.g. `Worldmaps/`, not sure what's up with `Worldmap-*/` and `Levels/`).

There is also no need to touch `repository/`. That folder should be considered a permanent storage location, whatever goes in there stays there forever, no changes or deletions.

**Comment by Hume2** [comment]
That's true. The `Worldmaps/` and `Worldmaps-0_3_5/` are really allocated in the `index` files, altrough there could be a link directly to the `repository/` directory. I guess it's intended, because of the old version compatibility, so let's let it as is.

The folder `Worldmaps-0_4_0/` is not allocated anywhere, I've created it because I thought that it's needed for something. I think we can delete it then.

The folder `Levels/` doesn't seem to be allocated form anywhere, at least I couldn't find it. It contains Cristoph's levels, those have been added to Bonus Island III and a development version of Levels of Wolfs, that is already finished and put somewhere else.

**Comment by maxteufel** [comment]
> The folder "translations/" is empty and therefore we can delete it.

It's a submodule which you most likely haven't checked out yet.

**Comment by Hume2** [comment]
> It's a submodule which you most likely haven't checked out yet.

That explains much.

**Comment by LMH0013** [comment]
Indeed the Worlmaps-*/ directories were meant for old version compatibility before the implementation of the repository/ directory.  What could (and probably should) be done is take the contents of Worldmaps/ and Worldmaps-*/ and move them into repository/.  Then all the filepaths in the all earlier index files needs to be correspondingly changed.  This should align the old content with the new organization without compromising the old version functionality.  

The Levels/ directory is also a relic from the original addon management architecture.  If I'm not mistaken, all of its contents have been moved elsewhere and should not be referenced or used by any version of SuperTux.  I think I kept them there to make sure I didn't break anything in the move but subsequently forgot to remove them.  Thus it should be fine to remove Levels/.
